language/de-DE/com_contact.ini

```
@@ -23,18 +26,44 @@ COM_CONTACT_COUNTRY="Land"
 COM_CONTACT_DEFAULT_PAGE_TITLE="Kontakte"
 COM_CONTACT_DETAILS="Kontakt"
 COM_CONTACT_DOWNLOAD_INFORMATION_AS="Informationen herunterladen als:"
+COM_CONTACT_EDIT_CONTACT="Kontact bearbeiten"
 COM_CONTACT_EMAIL_FORM="Kontaktformular"
 COM_CONTACT_EMAIL_LABEL="E-Mail"
 COM_CONTACT_EMAIL_THANKS="Danke für die E-Mail!"
 COM_CONTACT_ENQUIRY_TEXT="Dies ist eine Mailanfrage via %s von:"
 COM_CONTACT_ERROR_CONTACT_NOT_FOUND="Kontakt nicht gefunden!"
 COM_CONTACT_FAX="Fax"
 COM_CONTACT_FAX_NUMBER="Fax: %s"
+COM_CONTACT_FIELD_CREATED_BY_ALIAS_LABEL="Autoralias"
+COM_CONTACT_FIELD_INFORMATION_ADDRESS_LABEL="Adresse"
+COM_CONTACT_FIELD_INFORMATION_COUNTRY_LABEL="Land"
+COM_CONTACT_FIELD_INFORMATION_FAX_LABEL="Fax"
+COM_CONTACT_FIELD_INFORMATION_MISC_LABEL="Weitere Informationen"
+COM_CONTACT_FIELD_INFORMATION_MOBILE_LABEL="Mobil"
+COM_CONTACT_FIELD_INFORMATION_POSITION_LABEL="Position"
+COM_CONTACT_FIELD_INFORMATION_POSTCODE_LABEL="Postleitzahl"
+COM_CONTACT_FIELD_INFORMATION_STATE_LABEL="Bundesland"
+COM_CONTACT_FIELD_INFORMATION_SUBURB_LABEL="Ort"
+COM_CONTACT_FIELD_INFORMATION_TELEPHONE_LABEL="Telefon"
+COM_CONTACT_FIELD_INFORMATION_WEBPAGE_LABEL="Webseite"
```

```suggestion
COM_CONTACT_FIELD_INFORMATION_WEBPAGE_LABEL="Website"
```
@ThomasFinnern zur Info, wir hatten uns mal auf folgendes Naming committet.
Webseite = eine einzelne Webseite auf einer Website
Website = Ein Internetauftritt